patches/4D530A26 - Gears of Wars Judgement.patch

```diff
@@ -47,13 +47,12 @@ title_id = "4D530A26"
     [[patch.be32]]
         address = 0x827E3970
         value = 0x38800004
-    [[patch.be32]]                #nop finishing render pass from branching to temporal AA resolve
+    [[patch.be32]] # nop finishing render pass from branching to temporal AA resolve
```

Comment:
Sus whitespaces

Reply:
Should there be a space after comment blocks?

Reply:
It depends if the prior line of comment is behind a line of code then yea.
```
My_code ; my comment
        ; my comment 2
```

Reply:
This change is good.